test/test_meta.py

```diff
@@ -711,6 +711,7 @@ def run_meta_crossref(
 meta_function_device_expected_failures['cpu'] = {
     torch.native_batch_norm: {bf16, f16},
     torch._native_batch_norm_legit: {bf16, f16},
+    torch.ops.aten.batch_norm_with_update: {bf16, f16},
```

Comment:
Why expected failures?

Reply:
These xfails are inherited from `_native_batch_norm_legit`. If we want to reenable these tests I prefer to do them in a separate PR in the future since they are pre-existing failures